examples/scan_test.py

```diff
@@ -0,0 +1,142 @@
+"""
+Test assets attachments scanning
+"""
+
+from datetime import date, datetime, timedelta
+from json import dumps as json_dumps
+from os import environ
+from warnings import filterwarnings
+
+from archivist.archivist import Archivist
+from archivist.assets import BEHAVIOURS
+from archivist.errors import ArchivistError
+from archivist.proof_mechanism import ProofMechanism
+
+filterwarnings("ignore", message="Unverified HTTPS request")
+
+
+def scan_test(arch, datestring, scanned_expected=False):
+    """
+    Test asset creation if not exists - check attachment for scanned status.
+
+    Because we use create_if_not_exists the asset and attachments will persist.
+
+    The test checks the scanned timestamp and checks scanned status.
+    The first attachment should return OK after 24 hours and the second attachment
+    should return bad after 24 hours.
+
+    Args:
+
+         date (str): date string of the form 2022-03-01
+         scanned_expected (bool): whether the attachment is expected to have been scanned
+            - true for yesterday
+    """
+
+    ASSET_NAME = f"Scan test asset with 2 attachments - one bad or not scanned {datestring}"
+
+    print(f"##[debug]Checking asset {ASSET_NAME}")
```

Comment:
This '##[]' print syntax is specific to az pipelines. It makes for coloured and organised log output. Maybe worth mentioning that somewhere?